tests/third_party/cupy/math_tests/test_sumprod.py

```diff
@@ -609,7 +611,7 @@ def test_diff_2dim_with_append(self, xp, dtype):
     @pytest.mark.usefixtures("allow_fall_back_on_numpy")
     @testing.with_requires("numpy>=1.16")
     @testing.for_all_dtypes()
-    @testing.numpy_cupy_allclose()
```

Comment:
`tests/third_party/cupy/math_tests/test_floating.py::TestFloating::test_copysign_combination` is failed on Iris XE due to different type of result